packages/expo-constants/src/Constants.types.ts

```diff
@@ -55,13 +92,21 @@ export interface AppManifest extends ExpoConfig {
     tool?: string;
     [key: string]: any;
   };
-  bundleUrl: string;
+  bundleUrl?: string;
   debuggerHost?: string;
   mainModuleName?: string;
   logUrl?: string;
-  [key: string]: any;
```

Comment:
This seems like a potential breaking change for developers who use the `AppManifest` type